aten/src/ATen/native/mps/operations/Distributions.mm

```diff
@@ -62,12 +62,12 @@
       // FP16, FP32 and Int32 are the only data types supported for distributions on MPS backend.
       const MPSDataType inputDataType = [&] {
         // only for random_mps, we pass interval range of type int64_t
-        if (std::is_same<scalar_t, int64_t>::value)
+        if constexpr (std::is_same_v<scalar_t, int64_t>) {
           return MPSDataTypeInt32;
-        else
```

Comment:
I guess the else is not helpful here?

Reply:
Perhaps it's my stylistic choice(and muscle memory from older compiler which used to raise a warning when there were no default return value) at play here, but I prefer:
```c
 if (pred) {
   return x;
 }
 return y;
```
to
```c
if (pred) {
  return x;
} else {
  return y;
}
```